docs/features/software-catalog/descriptor-format.md

```diff
@@ -444,6 +445,19 @@ Apart from being a string, the software catalog leaves the format of this field
 open to implementers to choose. Most commonly, it is set to the ID or email of a
 group of people in an organizational structure.
 
+### `spec.system` [optional]
+
+An [entity reference](#string-references) that to the system that the component
+belongs to, e.g. `artist-engagement-portal`. This field is optional.
+
+| Default [`kind`](#apiversion-and-kind-required) | Default [`namespace`](#namespace-optional) | Generated [relation](well-known-relations.md) type                            |
+| ----------------------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------------- |
+| [`System`](#kind-system) (only)                 | Same as this entity, typically `default`   | [`partOf`, and reverse `hasPart`](well-known-relations.md#partof-and-haspart) |
+
+| [`kind`](#apiversion-and-kind-required)                   | Default [`namespace`](#namespace-optional) | Generated [relation](well-known-relations.md) type                            |
+| --------------------------------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------------- |
+| [`System`](#kind-system) (default), [`API`](#kind-system) | Same as this entity, typically `default`   | [`partOf`, and reverse `hasPart`](well-known-relations.md#partof-and-haspart) |
```

Comment:
@freben I made two possible table variants, if we settle on one I can take care to update all other places too.

First one is the one you suggested. I'm not sure if the first column "Default kind" is the right choice. This works well if you only have a single kind. But for owner we would have multiple kinds, so I would suggest to go with the second  table that is just "kind" and declares the default kind in the row below

Reply:
Right, agreed, it should say both what the possible values are and what the default is when left out. I'll look closer at a later time

Reply:
So I introduced this for all entity references and removed the previous text sections for them.

Reply:
Looking great